pyvo/auth/__init__.py

```diff
@@ -1 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+__all__ = ["AuthSession", "AuthURLs", "CredentialStore"]
+
+from .authsession import AuthSession
```

Comment:
I usually `import *` here so when new things are added in the submodule all I need to do is update `__all__` for that module.

Reply:
https://stackoverflow.com/questions/2386714/why-is-import-bad

Reply:
This is the only case when I ever use `import *`. The usual pitfalls do not apply here because  it only pulls what it is in the defined `__all__` of the modules and there is no code in `__init__`. 

I prefer `__all__` in the submodule as it's closer to the actual code and easier to get the context but I don't have a strong opinion.

Reply:
Mea culpa. In that case it should be fine, explicitness is ensured...